src-js/instantsearch.js

```diff
@@ -0,0 +1,76 @@
+import * as throttle from 'lodash.throttle';
+
+// Instant search initialization
+export function init () {
+  var search = instantsearch({
+    appId: 'U0RXNGRK45',
+    apiKey: 'dd49f5b81d238d474b49645a4daed322', // search-only API Key; safe for front-end code
+    indexName: 'documentation',
+    routing: true
+  });
+
+  // initialize SearchBox
+  search.addWidget(
+    instantsearch.widgets.searchBox({
+      container: '#search-box',
+      cssClasses: {
+        input: 'instantsearch-search'
+      },
+      placeholder: 'Search Documentation',
+      queryHook: throttle(function (query, searchFunction) { searchFunction(query); }, 1000) // method to throttle search requests
```

Comment:
We want to `debounce` not `throttle`, no? `throttle` should mean that the, if the user types too fast, the final search will be dropped. `debounce` means that the early searches will be dropped, and searching will wait until the input settles.

Reply:
After reading [this article](https://css-tricks.com/the-difference-between-throttling-and-debouncing/), I think @Peeja is right on this 😄

Reply:
Yep, looks like it.